ansible_collections/arista/avd/examples/campus-fabric/intended/configs/LEAF1A.cfg

```diff
@@ -0,0 +1,1078 @@
+   dot1x mac based authentication
+   dot1x timeout tx-period 3
+   dot1x timeout reauth-period server
+   dot1x reauthorization request limit 3
+   spanning-tree portfast
+   spanning-tree bpduguard enable
+!
+interface Ethernet45
+   description
+   no shutdown
+   switchport trunk native vlan 110
+   switchport phone vlan 120
+   switchport phone trunk untagged
+   switchport mode trunk phone
+   switchport
+   dot1x pae authenticator
+   dot1x authentication failure action traffic allow vlan 130
+   dot1x reauthentication
+   dot1x port-control auto
+   dot1x host-mode multi-host authenticated
+   dot1x mac based authentication
+   dot1x timeout tx-period 3
+   dot1x timeout reauth-period server
+   dot1x reauthorization request limit 3
+   spanning-tree portfast
+   spanning-tree bpduguard enable
+!
+interface Ethernet46
+   description
+   no shutdown
+   switchport trunk native vlan 110
+   switchport phone vlan 120
+   switchport phone trunk untagged
+   switchport mode trunk phone
+   switchport
+   dot1x pae authenticator
+   dot1x authentication failure action traffic allow vlan 130
+   dot1x reauthentication
+   dot1x port-control auto
+   dot1x host-mode multi-host authenticated
+   dot1x mac based authentication
+   dot1x timeout tx-period 3
+   dot1x timeout reauth-period server
+   dot1x reauthorization request limit 3
+   spanning-tree portfast
+   spanning-tree bpduguard enable
+!
+interface Ethernet47
+   description
+   no shutdown
+   switchport trunk native vlan 110
+   switchport phone vlan 120
+   switchport phone trunk untagged
+   switchport mode trunk phone
+   switchport
+   dot1x pae authenticator
+   dot1x authentication failure action traffic allow vlan 130
+   dot1x reauthentication
+   dot1x port-control auto
+   dot1x host-mode multi-host authenticated
+   dot1x mac based authentication
+   dot1x timeout tx-period 3
+   dot1x timeout reauth-period server
+   dot1x reauthorization request limit 3
+   spanning-tree portfast
+   spanning-tree bpduguard enable
+!
+interface Ethernet48
+   description
+   no shutdown
+   switchport trunk native vlan 110
+   switchport phone vlan 120
+   switchport phone trunk untagged
+   switchport mode trunk phone
+   switchport
+   dot1x pae authenticator
+   dot1x authentication failure action traffic allow vlan 130
+   dot1x reauthentication
+   dot1x port-control auto
+   dot1x host-mode multi-host authenticated
+   dot1x mac based authentication
+   dot1x timeout tx-period 3
+   dot1x timeout reauth-period server
+   dot1x reauthorization request limit 3
+   spanning-tree portfast
+   spanning-tree bpduguard enable
+!
+interface Ethernet51
+   description SPINE1_Ethernet1
+   no shutdown
+   channel-group 51 mode active
+!
+interface Ethernet53
+   description MLAG_PEER_LEAF1B_Ethernet53
+   no shutdown
+   channel-group 53 mode active
+!
+interface Ethernet54
+   description MLAG_PEER_LEAF1B_Ethernet54
+   no shutdown
+   channel-group 53 mode active
+!
+interface Management0
```

Comment:
since this is campus, I guess OOB is not too commmon, so I suggest to remove everything about vrf MGMT and ma0.